src/test/java/io/vlingo/http/resource/sse/SseFeedTest.java

```diff
@@ -54,11 +53,11 @@ public void testThatFeedFeedsMultipleSubscribers() {
     final SseSubscriber subscriber2 = new SseSubscriber("all", client, "ABC456", "42");
     final SseSubscriber subscriber3 = new SseSubscriber("all", client, "ABC789", "43");
 
-    context.channel.untilRespondWith = TestUntil.happenings(3);
+    context.channel.expectRespondWith(3);
 
     feed.to(Arrays.asList(subscriber1, subscriber2, subscriber3));
 
-    context.channel.untilRespondWith.completes();
+    context.channel.untilCompletes();
```

Comment:
Eliminate this line and replace with:
assertEquals(3, (int) respondWithSafely.readFrom("foo"));